### ISSUE TITLE
Sign used for member's loads (Point or Distributed)

### ISSUE THREAD

**Issue by LcnGnd** [JWock82/PyNite#13]
I'm not sure about member's local referring system and about sign to use to applicate member's loads. I read in FEModel3D.py that :

![Cattura](https://user-images.githubusercontent.com/56585383/69866693-1afe5200-12a5-11ea-87de-40192c1d233c.JPG)

But, in FixedEndReactions.py, about PtLoad, If I apply a load P in "Fy" direction at x=L/2 I obtain following fixed reactions:
Fyi= P/2
Fyj= P/2
Mzi= PL/8
Mzj= -PL/8

![Cattur1](https://user-images.githubusercontent.com/56585383/69867074-138b7880-12a6-11ea-92c9-d28dc8e59b03.JPG)

If load P is downwards, reactions Fyi and Fyj will be upwards and they would be opposite in sign.
So, which is the correct way to understand how to applicate loads and then read results?

**Comment by JWock82** [comment]
This has to do with the sign convention usually used for beams. Think of a typical floor beam laying in a horizontal (XZ) plane. We tend to think of positive loads as loads acting downward on the beam. Similarly, negative load would indicate uplift on the beam. Under this "beam sign convention", a positive load acts in the beams local negative y direction, since the local y-axis points upward. Therefore, using this convention, a positive (downward) load acting in the negative local-y direction produces positive (upward) reactions that act in the positive local-y direction at the ends of the beam.

It's a little confusing, but it makes sense if you think of loads acting toward the beam as positive. If you want further verification, I've tested it out in two of the simple beam examples in the 'Examples' folder, and the results are correct.

**Comment by LcnGnd** [comment]
So, according with your answer, this convention is valid also for loads act in the beams local z direction (transverse direction). Instead, different convention for axial loads is used.

Moreover, about outputs:
1.  Positive (downward) load acting in the negative local-y direction produces negative displacements that act in the negative local-y direction (downwards). Because of this convention displacements are plottend in negative part of graph

2. According to usual convention used for beams, positive bending moment should be rappresented with positive values downwards. I think changes in Member3D.PlotMoment are needed

**Comment by JWock82** [comment]
I think it depends on what software you're using. I have seen different commercial programs use different sign conventions for bending moments. The argument for having positive moments in the negative region is that it shows you where the rebar is needed for concrete members.

I've chosen this convention because it's convenient for the deflection plot. Generally, you want to see the deflected shape of the beam when you're looking at a displacement plot. To have a load applied downward yielding an upward curve in the displacement plot is confusing to the user.

I'll have a closer look at what various commercial packages are doing and then make a final decision.

**Comment by LcnGnd** [comment]
I think that sign convention for bending moment is an important aspect of the program. Infact, have positive moments in the negative region allow us to understand where tended fibres are and thisi is very  important in reinforced concrete structures. Convention used now in the code can create some confusion for the user. 
Also deflected shape is important as mach as bending moment. So, a load applied downward should create the displacement's curve in the negative region and bending moment curve in the negative region but with positive values.

Best regards

**Comment by JWock82** [comment]
I've looked into this a little more. I found a few interesting discussions on the subject here: 

- [https://www.eng-tips.com/viewthread.cfm?qid=226200](url)

- [https://www.eng-tips.com/viewthread.cfm?qid=453909](url)

I've compared it to a number of commercial packages, and it looks like the sign convention you've proposed is commonly used, especially for 3D structures. I'll make the switch in sign convention at some point, but it's going to require changing a lot of code, so it won't be a quick change.
